File: Animator/Java/src/Lib/Rotator.java
package Lib;

public class Rotator extends EffectBase<IRotatable> {
    private final double angle;

    public Rotator(double angle) {
        this.angle = angle;
    }

    @Override
    protected void doAnimation(IRotatable o) {
        o.setAngle(this.angle);
    }
}

File: Animator/C#/PatternsAnimator/Lib/Rotator.cs
namespace PatternsAnimator.Lib
{
    public class Rotator : EffectBase<IRotateable>
    {
        private readonly double _angle;

        public Rotator(double angle)
        {
            _angle = angle;
        }

        protected override void DoAnimation(IRotateable o)
        {
            o.SetAngle(_angle);
        }
    }
}